types/mapbox-gl/index.d.ts

```diff
@@ -73,7 +73,12 @@ declare namespace mapboxgl {
 
     type PluginStatus = 'unavailable' | 'loading' | 'loaded' | 'error';
 
-    type LngLatLike = [number, number] | LngLat | { lng: number; lat: number } | { lon: number; lat: number } | [number, number];
+    type LngLatLike =
```

Comment:
Sorry for being picky, but why has this been reformatted? Are you using the Prettier config that comes with this repo?

Reply:
@dmytro-gokun Nah. That line seems to be above the 120 character print limit. I think the problem already comes from a previous commit.

Reply:
I just ran `npm run lint mapbox-gl` but I just tried that again and that doesn't seem to make this change. I'll revert this change.

Reply:
Ah this is the what is responsible: `npm run prettier -- --write types/mapbox-gl/**/*.ts`

I'm far from an expert on what all this stuff does!

Reply:
Does it reformat and break it in multiple lines when running that?

If yes then I think it's fine, you don't need to remove it. Our settings clearly state that the maximum print length is 120 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fc1f5ad5889db0ffd1726f1d4a8b2761a848ba6b/.prettierrc.json#L7

And for what I see that type definition was a bit longer than 120.

I think the issue came from a previous PR and someone committed a change without running prettier. You're not to blame 🙂

Reply:
@amxfonseca You might be right, the issue was here before. As soon as the final code is formatted using DefinitelyTyped's prettier settings, it's fine with me :).